assembl/static2/js/app/components/debate/common/sentiments.jsx

```diff
@@ -54,22 +80,19 @@ const Sentiment = ({ sentimentCounts, mySentiment, sentiment, client, isSelected
                     post: {
                       id: postId,
                       sentimentCounts: {
-                        like:
-                            sentiment.camelType === 'like'
-                              ? sentimentCounts.like + 1
-                              : sentimentCounts.like - (mySentiment === 'LIKE' ? 1 : 0),
+                        like: sentiment.camelType === 'like' ? likeCount + 1 : likeCount - (mySentiment === 'LIKE' ? 1 : 0),
```

Comment:
These `LIKE`, `DISAGREE`, etc should all become enums in the types file. Understandably, this isn't the scope of this PR.

Reply:
I was about to do it but since those come from back-end and are used in `SentimentTypes` type in `graphql_types.flow.js`, I'm not sure we should do it